sipa/templates/usersuite/_index_status.html

```diff
@@ -102,7 +111,11 @@ <h2 id="status-information">{{ _("Statusinformationen") }}</h2>
                 {%- set style = ("text-{}".format(property.style) if property.style else '') -%}
                 <td class="col-md-3">
                     <span class="d-inline-flex w-100 justify-content-between">
-                        <span class="{{ style }}">{{ property.value }}</span>
+                        {% if property.displayable %}
+                            <span class="{{ style }}">{{ property.value }}</span>
+                        {% else %}
+                            <span class="{{ style }}"></span>
+                        {% endif %}
```

Comment:
what precisely is the point of introducing `displayable` to set it to `False` in the `sample` backend?
I'm not sure what this is supposed to achieve.

Reply:
otherwise it will try to display the clients which is not feasible cause of there amount. So I Tried to mitigate this by introducing the displayable. But forgot it in the pycorft backend